### PR TITLE
[T-6596][17.0][MIG + REF] reports_font_size: refactoring to v17

### PR DESCRIPTION
**Pull request by Tisho99**

### CODE REVIEW

**Comment by Tisho99** [comment]
@ValentinVinagre @HaraldPanten PR moved to this repo

**Comment by ValentinVinagre** [comment]
@sara-castello could u review please?

**Review by sara-castello — APPROVED** [review]
LGTM

**Comment by ValentinVinagre** [comment]
@manuelregidor could u review?
@Tisho99 could u review the conflicts.

File: reports_font_size/README.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+-  Go to Settings / General Settings / Companies, and click on the
+   'Configure Document Layout' button
+-  You can the text size, text size unit, header size, and header size
```
Comment (manuelregidor) [inline reports_font_size/README.rst]: What can you do?

File: reports_font_size/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "author": "Sygel, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "web",
+    ],
+    "data": {
+        "templates/report_templates.xml",
+        "views/sale_order_type_view.xml",
```
Comment (manuelregidor) [inline reports_font_size/__manifest__.py]: Wrong file name

**Review by manuelregidor — CHANGES_REQUESTED** [review]

**Comment by Tisho99** [comment]
@manuelregidor Can you review again?

File: reports_font_size/__manifest__.py (hunk 2 above)
Comment (ValentinVinagre) [reply reports_font_size/__manifest__.py]: @Tisho99 has it been solved? give an answer.

File: reports_font_size/README.rst (hunk 1 above)
Comment (ValentinVinagre) [reply reports_font_size/README.rst]: @Tisho99 has it been solved? give an answer.

File: reports_font_size/__manifest__.py (hunk 2 above)
Comment (Tisho99) [reply reports_font_size/__manifest__.py]: yes

File: reports_font_size/README.rst (hunk 1 above)
Comment (Tisho99) [reply reports_font_size/README.rst]: yes

**Review by manuelregidor — APPROVED** [review]
LGTM

**Comment by Tisho99** [comment]
@ValentinVinagre Can you merge?

**Comment by ValentinVinagre** [comment]
> @ValentinVinagre Can you merge?

well...well...I see...I will summon the magic power of the sy-bot!!!! 😈 🔥 🔥 Oh... bruixas and curuixas please make this PR go to the main branch correctly:

/ocabot merge nobump

**Comment by HaraldPanten** [comment]
> On my way to merge this fine PR! Prepared branch [17.0-ocabot-merge-pr-13-by-Tisho99-bump-nobump](https://github.com/sygel-technology/sy-server-backend/commits/17.0-ocabot-merge-pr-13-by-Tisho99-bump-nobump), awaiting test results.

Why did it take too long for the sygel-bot to merge this PR? @manuel-florido @ValentinVinagre 😝😝

**Comment by ValentinVinagre** [comment]
> > On my way to merge this fine PR! Prepared branch [17.0-ocabot-merge-pr-13-by-Tisho99-bump-nobump](https://github.com/sygel-technology/sy-server-backend/commits/17.0-ocabot-merge-pr-13-by-Tisho99-bump-nobump), awaiting test results.
> 
> Why did it take too long for the sygel-bot to merge this PR? @manuel-florido @ValentinVinagre 😝😝

... magic issues ...

**Comment by manuel-florido** [comment]
> > On my way to merge this fine PR! Prepared branch [17.0-ocabot-merge-pr-13-by-Tisho99-bump-nobump](https://github.com/sygel-technology/sy-server-backend/commits/17.0-ocabot-merge-pr-13-by-Tisho99-bump-nobump), awaiting test results.
> 
> Why did it take too long for the sygel-bot to merge this PR? @manuel-florido @ValentinVinagre 😝😝

Rusty bot, WD-40 needed...

**Pull request merged**